pom.xml

```diff
@@ -349,6 +349,11 @@
                             <includes>
                                 <include>src/*/java/**/*.java</include>
                             </includes>
+                            <ratchetFrom>origin/master</ratchetFrom> <!-- this means that the whole spottless check/apply will run only against differences to origin/master branch-->
```

Comment:
I don't think we want this as the comparison point. We want the changes going into the current commit.